include/okapi/impl/device/rotarysensor/IMU.hpp

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Please run clang-format (there is a script in the project root) and format documentation as full sentences.

Reply:
I have made this change, it will be present in the next commit